### ISSUE TITLE
Editoast can't stuck for http requests

### ISSUE THREAD

**Issue by bloussou** [OpenRailAssociation/osrd#9403]
### What happened?

You go on the map  with many objects, you scroll up and down and editoast will be stuck.

if you exec in the image you can docker `docker exec -it osrd-editoast /bin/sh`:

- `editoast healthcheck`: pass
- `localhost:8090/health`: timeout

### What did you expect to happen?

A normal behavior

### How can we reproduce it (as minimally and precisely as possible)?

Launch this script locally:
```python
import threading
import requests
from datetime import datetime

# Define the URL endpoint
URL = "http://localhost:8090/layers/tile/speed_sections/geo/5/16/11/?infra=2"

# Function that will be run by each thread
def call_endpoint():
    try:
        response = requests.get(URL, verify=False)
        print(f"{datetime.now()} Status Code: {response.status_code}, Response: {response.text}")
    except requests.RequestException as e:
        print(f"{datetime.now()} Error: {e}")

# Create a list to hold threads
threads = []

# Launch 10 threads

for i in range(10):
    for i in range(100):
        thread = threading.Thread(target=call_endpoint)
        threads.append(thread)
        thread.start()

    # Wait for all threads to complete
    for thread in threads:
        thread.join()

print("All threads have completed.")
```

### On which environments the bug occurs?

Dev (SNCF), Recette (SNCF), Integration (SNCF), Custom (Docker compose)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

698b8f4c1ee820da5ba5b8531d9dd1295066ffa7

**Comment by bloussou** [comment]
On my computer I got stuck with this state:
```
Tasks (166) BUSY Running (0) IDLE Idle (166)
Warn  ID  State  Name  Total- Busy   Sched  Idle   Polls Kind   Location                                                                                           Fields                                                                     
        9 IDLE          3m08s   98ms   15ms  3m08s 98    task   src/core/mq_client.rs:202:9                                                                        target=tokio::task
       12 IDLE          3m04s   25ms   26ms  3m03s 152   task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task
       56 IDLE          1m51s  238ms  243ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       57 IDLE          1m51s   33ms   87ms  1m51s 9     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       58 IDLE          1m51s  194ms  392ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       59 IDLE          1m51s  150ms  384ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       60 IDLE          1m51s  149ms  187ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       61 IDLE          1m51s  135ms  232ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       62 IDLE          1m51s  145ms  218ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       63 IDLE          1m51s  116ms  225ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       64 IDLE          1m51s  145ms  549ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       65 IDLE          1m51s  273ms  182ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       66 IDLE          1m51s  117ms  367ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       67 IDLE          1m51s  133ms  317ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       68 IDLE          1m51s  124ms  287ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       69 IDLE          1m51s  250ms  146ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       70 IDLE          1m51s  156ms  148ms  1m51s 19    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       71 IDLE          1m51s  183ms  152ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       84 IDLE          1m51s  258us   92ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       85 IDLE          1m51s  199ms  349ms  1m51s 19    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       86 IDLE          1m51s  315ms  176ms  1m51s 16    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       87 IDLE          1m51s  174ms  223ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       88 IDLE          1m51s  192ms  180ms  1m51s 16    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       89 IDLE          1m51s  257ms  198ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       90 IDLE          1m51s  210ms  424ms  1m51s 16    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       91 IDLE          1m51s  196ms  256ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       92 IDLE          1m51s  242ms  197ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       93 IDLE          1m51s  139ms  162ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       94 IDLE          1m51s  139ms  216ms  1m51s 18    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       95 IDLE          1m51s  175ms  331ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       97 IDLE          1m51s   82ms  209ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       98 IDLE          1m51s  181ms  184ms  1m51s 17    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
       99 IDLE          1m51s  141ms  296ms  1m51s 16    task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      108 IDLE          1m51s  345us   15ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      109 IDLE          1m51s  213us   15ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      110 IDLE          1m51s   76ms   15ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      111 IDLE          1m51s    8ms   66ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      112 IDLE          1m51s  790us   85ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      113 IDLE          1m51s  786us   86ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      114 IDLE          1m51s  608us   87ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      115 IDLE          1m51s  804us   87ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      116 IDLE          1m51s    2ms   61ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      117 IDLE          1m51s  628us   91ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      118 IDLE          1m51s   35ms   92ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      119 IDLE          1m51s    7ms  110ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      120 IDLE          1m51s  845us  109ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      121 IDLE          1m51s  936us  118ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      122 IDLE          1m51s  200us   10ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      123 IDLE          1m51s    3ms   51ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      124 IDLE          1m51s    2ms   53ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      125 IDLE          1m51s  359us   55ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      126 IDLE          1m51s  293us   55ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      127 IDLE          1m51s    4ms   56ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      128 IDLE          1m51s    1ms   60ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      129 IDLE          1m51s  549us   61ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      130 IDLE          1m51s    3ms   62ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      131 IDLE          1m51s  867us   65ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      132 IDLE          1m51s  211us   66ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      133 IDLE          1m51s  197us   66ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      134 IDLE          1m51s  177us   66ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      135 IDLE          1m51s  170us   66ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
      136 IDLE          1m51s  185us   67ms  1m51s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/axum-0.7.7/src/serve.rs:253:17      target=tokio::task
188 IDLE          3m17s    2ms   53us  3m17s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         189 IDLE          3m16s    5ms  283ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         190 IDLE          3m16s   11ms  448ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         191 IDLE          3m16s    6ms  429ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         192 IDLE          3m16s  408us  287ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         193 IDLE          3m16s  462us  185ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         194 IDLE          3m16s    8ms  202ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         195 IDLE          3m16s  784us   63ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         196 IDLE          3m16s   11ms  200ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         197 IDLE          3m16s    3ms   33ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         198 IDLE          3m16s  352us   73ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         199 IDLE          3m16s  742us   25ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         200 IDLE          3m16s  593us   81ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         203 IDLE          3m16s  395us   37ms  3m16s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         204 IDLE          3m16s   19ms    8ms  3m16s 9     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         205 IDLE          3m16s  319us   68ms  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         207 IDLE          3m16s  589us    3ms  3m16s 9     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         208 IDLE          3m16s  353us   27ms  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         209 IDLE          3m15s    5ms    4ms  3m15s 10    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         210 IDLE          3m15s   16ms   33ms  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         211 IDLE          3m15s    2ms    6ms  3m15s 10    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         212 IDLE          3m15s    1ms   27ms  3m15s 11    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         216 IDLE          3m15s  480us    3ms  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         217 IDLE          3m15s  101us   10us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         218 IDLE          3m15s  101us    5ms  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         219 IDLE          3m15s    1ms    3ms  3m15s 10    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         221 IDLE          3m15s   89us    5ms  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         222 IDLE          3m15s   72us  197us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         223 IDLE          3m15s   91us    9us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         226 IDLE          3m15s   10ms    9ms  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         229 IDLE          3m15s    4ms   68us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         233 IDLE          3m15s  435us    3ms  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         234 IDLE          3m15s  244us   12us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         236 IDLE          3m15s    3ms  148us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         237 IDLE          3m15s  184us    9us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         239 IDLE          3m15s   93us  649us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         240 IDLE          3m15s   88us  960us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         241 IDLE          3m15s   80us    8us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         242 IDLE          3m15s  125us   11us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         243 IDLE          3m15s    1ms  910us  3m15s 10    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         244 IDLE          3m15s    4ms    0ns  3m15s 1     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         246 IDLE          3m15s   95us   19us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         249 IDLE          3m15s  287us   13us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         250 IDLE          3m15s  214us    7us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         251 IDLE          3m15s  299us    4ms  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         253 IDLE          3m15s  554us    1ms  3m15s 11    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         255 IDLE          3m15s   55us    6us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         256 IDLE          3m15s  108us    7us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         257 IDLE          3m15s  558us    1ms  3m15s 9     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         258 IDLE          3m15s    3ms  972us  3m15s 10    task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         260 IDLE          3m15s    4ms  747us  3m15s 8     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
         261 IDLE          3m15s   85us    6us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         263 IDLE          3m15s   76us   10us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         265 IDLE          3m15s   79us    8us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         267 IDLE          3m15s   95us   92us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
         269 IDLE          3m15s  101us  509us  3m15s 2     task   /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/redis-0.25.4/src/aio/tokio.rs:178:9 target=tokio::task                                                      
>>       271 IDLE          3m15s  338us  908us  3m15s 9     task   editoast_models/src/db_connection_pool.rs:446:9                                                    target=tokio::task                                                      
```

**Comment by flomonster** [comment]
fixed by #9420
